src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { profile } from "../../assets";
+import "./nav.css";
+
+function Nav() {
+  return (
+    <div>
+      <div className="navbar">
+        <div className="logo-container">
+          <div>
+            Designing<span className="theme-color">Wise</span>
+          </div>
+        </div>
+        <div className="menu-container">
+          <ul className="menu-list">
+            <li className="menu-list-item">
+              <Link to="/" className="active">Home</Link>
+            </li>
+            <li className="menu-list-item">
+              <Link to="/videoListing">Explore</Link>
+            </li>
+            <li className="menu-list-item">
+              <Link to="/playlist">Playlist</Link>
+            </li>
+            <li className="menu-list-item">
+              <Link to="/watchLater">Watch Later</Link>
+            </li>
+            <li className="menu-list-item">
+              <Link to="/history">History</Link>
```

Comment:
can we create object for these list , and map over it

Reply:
not sure, because these are just button of single page video, so I don't think so that it is required